resources/dicts/relationship_events/normal_interactions/trust/decrease.json

```diff
@@ -1,207 +1,237 @@
+			"r_c forgot something that m_c asked {PRONOUN/r_c/object} to remember.",
+			"r_c won't stop looking at a specific Clanmate while m_c talks to {PRONOUN/r_c/object}.",
+			"m_c has a weird feeling about something r_c said the other day."
+		]
+	},
+	{
+		"id": "trust_de_low2",
+		"intensity": "low",
+		"interactions": [
+			"m_c shared a embarrassing story about r_c {PRONOUN/m_c/subject} {VERB/m_c/were/was} asked not to share...",
+			"m_c's friend doesn't trust r_c.",
+			"m_c learns that r_c told a Clanmate one of {PRONOUN/m_c/poss} secrets.",
+			"r_c snitches on m_c for something minor.",
+			"m_c saw r_c hide something before talking to {PRONOUN/m_c/object}.",
+			"r_c won't tell m_c what {PRONOUN/r_c/subject} ate this morning.",
+			"r_c gets a strange expression when m_c walks up to ask a question.",
+			"m_c has to snap r_c out of a daze to ask a question.",
+			"m_c subtly accuses r_c of being untruthful about something.",
+			"r_c keeps asking m_c if {PRONOUN/m_c/subject} {VERB/m_c/trust/trusts} {PRONOUN/r_c/object}.",
+			"m_c finds out something small about r_c that r_c had lied about before."
+
+		],
+		"reaction_random_cat": {
+			"trust": "decrease"
+		}
+	},
+	{
+		"id": "trust_de_med1",
+		"interactions": [
+			"m_c knows that r_c just lied!",
+			"r_c gave m_c a gift that doesn't make sense.",
+			"r_c forgot something important about m_c.",
+			"r_c keeps calling m_c by the wrong name.",
+			"r_c sides with a Clanmate instead of m_c on something.",
+			"m_c thinks that r_c spread a rumor about {PRONOUN/m_c/object}.",
+			"m_c doesn't think that r_c's intentions are all pure.",
+			"m_c keeps thinking {PRONOUN/m_c/subject} {VERB/m_c/see/sees} r_c sneaking around the camp at night.",
+			"r_c keeps making excuses about something m_c wants to know.",
+			"m_c can't seem to get a straight answer from r_c lately.",
+			"r_c let m_c down while m_c was counting on {PRONOUN/r_c/object}."
+		]
+	},
+	{
+		"id": "trust_de_med2",
+		"interactions": [
+			"m_c said {PRONOUN/m_c/subject}'d cover for r_c, but when it came time, {PRONOUN/m_c/subject} got r_c in trouble.",
+			"r_c doesn't want to apologize to m_c for something.",
+			"r_c asks m_c to stop following {PRONOUN/r_c/object} around.",
+			"r_c keeps avoiding eye contact with m_c.",
+			"m_c learns that r_c has been misleading {PRONOUN/m_c/object} about something trivial.",
+			"r_c does something that badly affects m_c.",
+			"r_c agrees to something for m_c that {PRONOUN/m_c/object} didn't want to do...",
+			"m_c and r_c can't seem to agree on the truth about something.",
+			"m_c asks r_c about a lie r_c told, but {PRONOUN/r_c/subject} {VERB/r_c/dodge/dodges} the question.",
+			"m_c realizes that r_c's been avoiding {PRONOUN/m_c/object}.",
+			"m_c knows that r_c is hiding something.",
+			"m_c was convinced that r_c was lying about something, but r_c wasn't.",
+			"The Clan realizes that m_c doesn't ask for help from r_c much lately."

```

Comment:
maybe instead of the entire clan, could have it be a Clanmate asking m_c why?

Reply:
ended up removing this since I liked it in concept but it read pretty clunky no matter how I reworded it!